pkg/operation/botanist/component/interfaces.go

```diff
@@ -47,6 +47,17 @@ type MigrateWaiter interface {
 	WaitMigrate(ctx context.Context) error
 }
 
+// MonitoringComponent exposes configuration for Prometheus as well as the AlertManager.
+type MonitoringComponent interface {
+	// ScrapeConfig returns the scrape configuration for Prometheus.
+	ScrapeConfig() ([]string, error)
```

Comment:
Shouldn't this rather be named `ScrapeConfigs`?
```suggestion
	ScrapeConfigs() ([]string, error)
```